components/match2/wikis/rainbowsix/match_group_input_custom.lua

```diff
@@ -342,6 +343,16 @@ function matchFunctions.getVodStuff(match)
 end
 
 function matchFunctions.getExtraData(match)
+	local casters = {}
+	for key, name in Table.iter.pairsByPrefix(match, 'caster') do
+		table.insert(casters, CustomMatchGroupInput._getCasterInformation(
+			name,
+			match[key .. 'flag'],
+			match[key .. 'name']
+		))
+	end
+	table.sort(casters, function(c1, c2) return c1.displayName:lower() < c2.displayName:lower() end)
```

Comment:
Why sort them when doing `pairs` in the display?

Reply:
I dont have a good answer for that. Mostly this is based off the Rocket League implementation, so if there is a better way to do this im all ears.